docs/examples/geopackage-json-ld.md

```diff
@@ -0,0 +1,118 @@
+# Schema.org Example Collection of Geopackage's
+
+Example metadata listing geopackage files using the [Schema.org](https://schema.org) vocabulary in
+`json+ld` format. See [#3](https://github.com/I-GUIDE/data-catalog/issues/3) and
+[#4](https://github.com/I-GUIDE/data-catalog/issues/4), for discussion that led to the creation of
+this example.
+
+```json
+{
+    "@context": "https://schema.org/",
+    "@type": "Dataset",
+    "url": "https://www.hydroshare.org/resource/3295a17b4cc24d34bd6a5c5aaf753c50",
+    "name": "geoconnex reference features",
+    "description": "These GeoPackage artifacts are for use in https://reference.geoconnex.us For more information see: https://github.com/internetofwater/geoconnex.us/tree/master/namespaces/ref/",
+    "creator": [
+        {
+            "@type": "Organization",
+            "email": "cgsinternetofwater@gmail.com",
+            "name": "Internet of Water Initiative, Center for Geospatial Solutions, Lincoln Institute of Land Policy",
+        }
+    ],
+    "provider": {
+        "@id": "https://www.hydroshare.org",
+        "@type": "Organization",
+        "name": "HydroShare",
+        "url": "https://www.hydroshare.org"
+    },
+    "distribution": {
+        "@type": "DataDownload",
+        "contentSize": "3.5 GB",
```

Comment:
According to SchemaOrg, contentSize should be expressed in "(mega/kilo) bytes". I don't like this because it won't scale for very small or very large files.

Reply:
I don't think this comment needs to hold up the PR merge so I created a ticket to continue the discussion: #7

Reply:
Yeah, I think this really comes down to the responsibility of the catalog. I could see benefits to having `contentSize` specified as the number of bytes like its specified in an HTTP header. However, I am fine with moving the conversation to #7 and leaving this as a todo once a decision has been made.